packages/storage/src/implementation/constants.ts

```diff
@@ -18,15 +18,17 @@
  * @fileoverview Constants used in the Firebase Storage library.
  */
 
+export const domain: string = "firebasestorage.googleapis.com";
```

Comment:
minor, but this should probably be called something like `api_base_url`, domain is a little ambiguous

Reply:
I picked `DOMAIN_NAME`, `API_BASE_URL` is taken below